accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-media-rules.ts

```diff
@@ -136,6 +146,8 @@ let a11yRulesMedia: Rule[] = [
             "dom:td[bgcolor]",
         run: (context: RuleContext, options?: {}): RuleResult | RuleResult[] => {
             const ruleContext = context["dom"].node as Element;
+            //skip the rule
+            if (RPTUtil.isNodeHidden(ruleContext)) return null;
```

Comment:
This is about color for something displayed. I don't think it should be skipped.

Reply:
removed